docs/framework/01intro/introduction.md

```diff
@@ -1,33 +1,26 @@
 # Introduction
+With several layer-one protocols gaining meaningful traction over the last couple of years and layer-two protocols taking [center stage in the scalability roadmap for Ethereum](https://vitalik.ca/general/2021/12/06/endgame.html), it is increasingly becoming clear that the future will be multichain. This thesis is further reinforced by emerging patterns such as modular blockchains, application-specific chains, and fractal scaling. A multichain world is one in which disparate chains co-exist with thriving ecosystems. 
 
-Risk is the measure of the extent to which an entity is threatened by potential circumstances or events, and is typically a function of (i) the adverse impacts that would arise if the circumstance or event occurs, and (ii) the likelihood of occurrence.  [NIST - National Institute of Standards and Technology]
+In the absence of a countervailing force, the proliferation of chains would naturally lead to increased fragmentation and the siloing of ecosystems. Cross-chain protocols directly address this problem by enabling interoperation across distinct chains. They reduce fragmentation, enable scalability, and improve market efficiency. In addition, such protocols potentially open up new design spaces for novel cross-chain native applications and use cases. Hence, cross-chain protocols have emerged as a crucial building block enabling the multichain future. 
```

Comment:
terminology: In the Q&A, the term cross-chain is crosschain. The rationale is, we don't have block chains or block-chains; we have blockchains.

Reply:
Sounds reasonable ... fixed